spec/features/courses/preview_spec.rb

```diff
@@ -160,9 +160,18 @@
 
     expect(preview_course_page).to_not have_salary_details
 
-    expect(preview_course_page.scholarship_amount).to have_content("a scholarship of £2,000")
-
-    expect(preview_course_page.bursary_amount).to have_content("a bursary of £4,000")
+    # START: finanical support placeholder
+    # NOTE: This is temporary.
```

Comment:
Is this more accurate? Might help any devs who aren't familiar with rollover better understand what's going on.

```suggestion
    # NOTE: This is temporary as the financial incentives for this cycle haven't been released yet.
```